test/e2e/play_kube_test.go

```diff
@@ -4515,6 +4515,26 @@ ENV OPENJ9_JAVA_OPTIONS=%q
 		})
 	})
 
+	It("podman play kube initContainers inherit environment", func() {
+		ctr := getCtr()
+		// the environment variable $container is set by
+		// generate.CompleteSpec, which must be called on initContainers
+		// as well. If it is not, then initCtr will fail
+		initCtr := getCtr(
+			withImage(ALPINE),
+			withCmd([]string{"sh", "-c", "\"[ $container = 'podman' ]\""}),
```

Comment:
As the poor schmoe who has to look at test failures, I reeeeeeeeally hate this kind of test. Here's why:
```
Sample failure:
     test failed! Exit status was nonzero!
```
...and, for comparison, here is what makes me wiggle my toes with joy:
```
    test failed: '$container' variable in container is undefined
```
or
```
    test failed: `$container` variable in container is 'something-surprising' (expected it to be 'podman')
```
That latter one is (probably) impossible IRL, so, yes, the test as it is will _almost_ certainly show a failure to set envariable... but if you make a general practice of show-the-value type of testing, every so often you will get a surprise failure and the test logs will show exactly what happened and you will have a fix ready in minutes instead of hours.

And, I write this in the spirit of appreciation (you fixed a bug! you took the time to write a test! Then you wrote MORE tests!). So, thank you!